package.json

```diff
@@ -48,8 +52,10 @@
     "chai-enzyme": "^1.0.0-beta.1",
     "clean-css": "^4.2.1",
     "create-react-class": "^15.6.3",
+    "css-loader": "^3.2.0",
     "enzyme": "^3.10.0",
-    "enzyme-adapter-react-helper": "^1.3.6",
+    "enzyme-adapter-react-16": "^1.14.0",
```

Comment:
this shouldn’t be needed what with the helper

Reply:
@ljharb `enzyme-adapter-react-16` is the same as this call `enzyme-adapter-react-install 16`?

Reply:
yep! it installs the right one for the version, and it’s important for testing of multiple react versions that no specific adapter is explicitly in dev deps.